components/ingest-service/chef-load-data/ohai-data.json

```diff
@@ -935,50 +935,6 @@
       "string_1": "Xen"
     }
   },
-  "ec2": {
```

Comment:
This is removed to allow the chef load to create multiple nodes in the nodemanager. The nodemanager checks if the ec2 IDs are different for each node.